datahub-web-react/src/graphql/chart.graphql

```diff
@@ -2,6 +2,7 @@ query getChart($urn: String!) {
     chart(urn: $urn) {
         urn
         type
+        exists
```

Comment:
do we need to add this to dashboard.graphql as well?

Reply:
(i see its in fragments)

Reply:
yup I got it in `dashboardFields` in fragments.graphql!